modin/pandas/series.py

```diff
@@ -168,6 +168,8 @@ def _getitem(self, key):
                     pandas.RangeIndex(len(self.index))[key]
                 )
             )
+        if isinstance(key, tuple):
```

Comment:
This is a good temporary solution but, in the future, we should have a better way of indexing series. Can we add a TODO here and open an issue pointing to this?